src/components/Pages/TrainingSeries/List/Tab/Tab.js

```diff
@@ -59,7 +58,7 @@ function Tab({
 
                     <Select
                       onChange={e => {
-                        selectedId = parseInt(e.target.value);
+                        tsMessages[0].id = parseInt(e.target.value);
                       }}
```

Comment:
Added training series to the and it was rendered into the home page listed under training-series.